src/app/downloads/downloads.component.ts

```diff
@@ -139,7 +140,7 @@ export class DownloadsComponent implements OnInit, OnDestroy {
           this.error = 'An error occurred while requesting podcast metrics on ' + this.filter.podcast.title;
           console.error(err);
         } else {
-          this.error = this.filter.podcast.title + ' podcast has no download metrics.';
+          this.error = this.filter.podcast.title + ' podcast has no download metrics for some or all of this time.';
```

Comment:
This change to the error message seems inaccurate, but the first attempt at providing an error message wasn't accurate either. If Castle doesn't have metrics for all or part of the time, it will populate the rest of the dates with zeroes (this was not initially true.) When I'm seeing these it's other errors occurring in Castle (that should be investigated.) I've also noticed that half assed attempt at showing these errors only supports one error at a time. Ticket?

Reply:
ticket sounds good to me -- will nix the changed error message for now. (in dev, i was getting 'has no download metrics' for stuff that did have data for part of the time, going off latest castle)

Reply:
Opened a ticket: https://github.com/PRX/metrics.prx.org/issues/53